src/devices/machine/roc10937.cpp

```diff
@@ -171,8 +171,8 @@ void rocvfd_device::device_reset()
 	m_shift_count = 0;
 	m_shift_data = 0;
 	m_pcursor_pos = 0;
-	m_count=0;
-	m_duty=31;
+	m_count = 0;
+	m_duty = 0;
 	m_disp = 0;
```

Comment:
How many systems is this going to affect?  What was the basis for the previous reset value, and what’s the rationale for changing it?

Reply:
The main reason for this adjustment is to recreate the behaviour of the native chip, as reported in the datasheet
https://pdf1.alldatasheet.com/datasheet-pdf/view/11031/OKI/MSC1937-01.html

In the past, we defaulted to full brightness on reset, based on the behaviour of the NEC starburst component used by BFM, of which this bears no resemblance. This is incorrect, and bugged behaviour, and games such as m4dtyfre exploit the new behaviour. Recently as part of a review task that brought me back to these drivers, I have attempted to source documentation for all components, and immediately noticed this issue.

As one example of the wrong behaviour, a common practice for doing a quick flashing effect in Barcrest games is to reset the MCU to clear the buffers and turn the display off, repopulate it with the new characters and toggle duty between full and 0 to get an instant appearance without flicker. In Duty Free you can see that the apparent fade to bring in the words DUTY FREE doesn't work, the characters start to appear ahead of time.

I believe a number of programs outside of FME use this, or the Samsung clones that share a basic MCU program, but in my tests I couldn't see any program that wasn't already setting duty to the correct value on reset, coping with the correct behaviour, although I admit I didn't regression test all of them, just a title per driver.

To that end, there are a lot of MAME driver layouts that are not honouring the use of the duty to add fading etc, this is probably something I will review after this commit.